src/PrestaShopBundle/Form/Admin/Sell/Product/Image/ImageDropzoneType.php

```diff
@@ -76,6 +76,10 @@ public function configureOptions(OptionsResolver $resolver)
                 'window.delete' => $this->trans('Delete selection', 'Admin.Actions'),
                 'window.close' => $this->trans('Close window', 'Admin.Actions'),
                 'window.zoom' => $this->trans('Zoom on selection', 'Admin.Actions'),
+                'modal.close' => $this->trans('Cancel', 'Admin.Actions'),
+                'modal.accept' => $this->trans('Delte', 'Admin.Actions'),
+                'modal.title' => $this->transChoice('Are you sure you want to delete the selected image?|Are you sure you want to delete the %filesNb% selected images?', 'Admin.Catalog.Notification'),
+                'delete.success' => $this->trans('The selection was successfully deleted.', 'Admin.Notifications.Success'),
```

Comment:
```suggestion
                'delete.success' => $this->trans('The selection has been successfully deleted.', 'Admin.Notifications.Success'),
```
Sorry the existing string is 'has been' and not 'was'

Reply:
Np, I failed myself writing "Delete" 😥